src/Utils/functions.jsx

```diff
@@ -46,12 +46,12 @@ export function blueToYellow(numColors) {
 }
 
 export function rawTextToShow(text) {
-  text = text.replace(/[-_]/g, " ");
-  const words = text.split(" ");
-  const capitalizedWords = words.map(
+  text = text?.replace(/[-_]/g, " ");
+  const words = text?.split(" ");
+  const capitalizedWords = words?.map(
     (word) => word.charAt(0).toUpperCase() + word.slice(1)
```

Comment:
I recommend extracing this function in another place, specially if you'll use it somewhere else

Reply:
but.. this is the somewhere else... where i extracted all functions to..